lib/timeline/Stack.js

```diff
@@ -304,7 +322,7 @@ function performStacking(items, margins, compareTimes, shouldStack, shouldOthers
       horizontalOverlapEndIndex = findIndexFrom(itemsAlreadyPositioned, i => itemEnd < getItemStart(i) - EPSILON, Math.max(horizontalOverlapStartIndex, horizontalOverlapEndIndex));
     }
     if(previousEnd !== null && previousEnd - EPSILON > itemEnd) {
-      horizontalOverlapEndIndex = findLastIndexBetween(itemsAlreadyPositioned, i => itemEnd + EPSILON >= getItemStart(i), horizontalOverlapStartIndex, horizontalOVerlapEndIndex) + 1;
+      horizontalOverlapEndIndex = findLastIndexBetween(itemsAlreadyPositioned, i => itemEnd + EPSILON >= getItemStart(i), horizontalOverlapStartIndex, horizontalOverlapEndIndex) + 1;
```

Comment:
There is a dedicated PR open for this typo :) https://github.com/visjs/vis-timeline/pull/1610